experimental/dds/tree/src/Common.ts

```diff
@@ -56,6 +56,13 @@ export function compareFiniteNumbers<T extends number>(a: T, b: T): number {
 	return a - b;
 }
 
+/**
+ * A numeric comparator used for sorting in descending order.
```

Comment:
Can you make this comment the same as the other non-reversed one (and maybe simplify it while you're at it?)